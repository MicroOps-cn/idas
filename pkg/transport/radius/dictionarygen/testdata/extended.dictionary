ATTRIBUTE SingleByteAttr 104 byte
