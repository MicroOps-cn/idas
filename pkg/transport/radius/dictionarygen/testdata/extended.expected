Constants:
	const (
		SingleByteAttr_Type radius.Type = 104
	)
Functions:
		func SingleByteAttr_Add(p *radius.Packet, value byte) (err error)
		func SingleByteAttr_Del(p *radius.Packet)
		func SingleByteAttr_Get(p *radius.Packet) (value byte)
		func SingleByteAttr_Gets(p *radius.Packet) (values []byte, err error)
		func SingleByteAttr_Lookup(p *radius.Packet) (value byte, err error)
		func SingleByteAttr_Set(p *radius.Packet, value byte) (err error)
